tracer/build/_build/Build.GitHub.cs

```diff
@@ -109,41 +109,58 @@ partial class Build
                 var changes = GitTasks.Git("diff master -- tracer/test/snapshots")
                                    .Select(f => f.Text);
 
-
                 const int minFiles = 50;
                 var nbSnapshotsModified = changes.Count(f => f.Contains("@@ "));
                 if (nbSnapshotsModified < minFiles)
                 {
                     // Dumb early exit, if we modify less than 50 files, we can review them manually
                     // Also it's certainly an addition of snapshot tests, so may not make sense to print a summary.
-                    Console.WriteLine("${nbSnapshotsModified} snapshots modified. Not doing snapshots diff for PRs modifying less than ${minFiles}.");
+                    Console.WriteLine($"{nbSnapshotsModified} snapshots modified. Not doing snapshots diff for PRs modifying less than {minFiles}.");
                     return;
                 }
 
                 const string unlinkedLinesExplicitor = "[...]";
+                var crossVersionTestsNamePattern = new [] {"VersionMismatchNewerNugetTests"};
```

Comment:
I didn't know that this NuGet mismatch was a thing :)

Reply:
Yes we can have auto instrumentation ran with one version of the assembly and custom version with another version. It has been a real issue for a while but the team mostly fixed it by doing some syncing/dark magic behind the scenes to make it work.
We still try to test for this use case in some scenarios where we could have issues